www/front_src/src/Resources/Graph/Performance/TimePeriods/useTimePeriod.tsx

```diff
@@ -206,9 +224,32 @@ const useTimePeriod = ({
         start: start.toISOString(),
         end: end.toISOString(),
       },
+      graphOptions: defaultGraphOptions,
     });
   };
 
+  React.useEffect(() => {
+    if (isNil(selectedTimePeriod) || isNil(details)) {
+      return;
+    }
+
+    if (not(preventQueryParametersUpdate.current)) {
```

Comment:
In what circumstance do we want to prevent the update of the queryParameters?

Reply:
To avoid display the loader when we update the listing that update the panel and performance graph.

Reply:
@bdauria Uh nevermind, it avoids the query parameters to update on more time when the hook is mounted

Reply:
To mimic the `componentDidUpdate`